packages/firestore/package.json

```diff
@@ -27,9 +25,9 @@
     "lint": "eslint -c .eslintrc.js '**/*.ts' --ignore-path '../../.gitignore'",
     "lint:fix": "eslint --fix -c .eslintrc.js '**/*.ts' --ignore-path '../../.gitignore'",
     "prettier": "prettier --write '*.js' '*.ts' '@(lite|exp|src|test)/**/*.ts'",
-    "pregendeps:exp": "yarn api-report:exp && node scripts/build-bundle.js --input ./exp/index.ts --output ./dist/exp/tmp.js",
+    "pregendeps:exp": "yarn api-report&& node scripts/build-bundle.js --input ./exp/index.ts --output ./dist/exp/tmp.js",
```

Comment:
Space before && (don't think it NEEDS it but for formatting)

Reply:
Done